swagger/swagger-invocation/invocation-core/src/main/java/io/servicecomb/swagger/invocation/arguments/ArgumentsMapperFactory.java

```diff
@@ -99,10 +99,12 @@ protected boolean isSwaggerWrapBody(ArgumentsMapperConfig config, List<ProviderP
       return false;
     }
 
-    swaggerType = ((Class<?>) swaggerType).getFields()[0].getGenericType();
-    Converter converter = converterMgr.findConverter(type, firstProviderParam, swaggerType);
-    if (ConverterCommon.class.isInstance(converter)) {
-      return false;
+    if (((Class<?>) swaggerType).getFields().length > 0) {
```

Comment:
must have a field in this place
if there is no field, that means it's bug, MUST not think this is a normal scenes.

Reply:
has remove this modify